app/controllers/users_controller.rb

```diff
@@ -0,0 +1,28 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
You have that already in ApplicationController, so you do not  need that  here :)